htdocs/adherents/list.php

```diff
@@ -435,7 +435,7 @@
 if ($sall)
 {
 	foreach ($fieldstosearchall as $key => $val) $fieldstosearchall[$key] = $langs->trans($val);
-	print '<div class="divsearchfieldfilter">'.$langs->trans("FilterOnInto", $sall).join(', ', $fieldstosearchall).'</div>';
+	print '<div class="divsearchfieldfilter">'.$langs->transnoentities("FilterOnInto", $sall).join(', ', $fieldstosearchall).'</div>';
```

Comment:
The translation string and the char to search may be special string. So we should html encode for HTML output.
Why removing ?

Reply:
https://github.com/Dolibarr/dolibarr/issues/15546